Tests/conf.json

```diff
@@ -3206,6 +3206,10 @@
             "fromversion": "6.0.0",
             "toversion": "6.0.9",
             "timeout": 250
+        },
+        {
+            "integrations": "FraudWatch",
+            "playbookID": "FraudWatch-Test"
```

Comment:
Why we need this?

Reply:
@ShahafBenYakir demisto-sdk validate requires me to write the test playbook here

Reply:
Then open an issue with steps to reproduce as it is not needed and remove from here.

Reply:
@ShahafBenYakir yes, same error
Packs/FraudWatch/Integrations/FraudWatch/FraudWatch.yml: [CJ102] - The following integration is not registered in Tests/conf.json file.
Please add:
{
    "integrations": "FraudWatch",
    "playbookID": "<TestPlaybook ID>"
}
to Tests/conf.json path under 'tests' key.
If you don't want to add a test playbook for this integration, please add: 
tests:
- No tests
to the file Packs/FraudWatch/Integrations/FraudWatch/FraudWatch.yml or run 'demisto-sdk format -i Packs/FraudWatch/Integrations/FraudWatch/FraudWatch.yml'

do you want me to write - No tests? or keep 
            "integrations": "FraudWatch",
            "playbookID": "FraudWatch-Test"
           in the test conf